package.json

```diff
@@ -50,8 +51,9 @@
   "devDependencies": {
     "@checkdigit/prettier-config": "^4.0.0",
     "@checkdigit/typescript-config": "^4.0.0",
-    "@types/jest": "^29.5.2",
-    "jest": "^29.6.0",
+    "@types/jest": "^29.5.3",
+    "fclone": "^1.0.11",
```

Comment:
another one. didn't realize why you were not using `structuredClone()` until I saw the usage of `fclone`